elasticsearch/es_index/aether-demo.json

```diff
@@ -0,0 +1,30 @@
+{
+	"mappings": {
+		"_doc": {
+			"_meta": {
+				"aet_geopoint": "geo_point",
+				"aet_join_field": "aet_relationship",
+				"aet_subscribed_topics": ["Person", "Building", "Household", "Survey"],
+				"aet_parent_field": {
+					"Person": "household_id",
+					"Household": "location_id",
+					"Building": "survey_id"
+				}
+			},
+			"properties": {
+				"geo_point": {
+					"type": "geo_point"
+				},
+				"aet_relationship": {
+					"type": "join",
+					"relations": {
+						"Survey": "Building",
+						"Building": "Household",
+						"Household": "Person"
+					}
+				}
+			}
+
+		}
+	}
+}
```

Comment:
More tabs?!!! :stuck_out_tongue:

Reply:
I have no idea why this happens :sob: I only use spaces!